modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -22,6 +22,14 @@ import {SidebarContext} from '../sidebar/SidebarContext.es';
 export default ({data, field, totalEntries, type}) => {
 	const {toggleSidebar} = useContext(SidebarContext);
 
+	const formatDate = (field) => {
+		moment.locale(themeDisplay.getLanguageId().split('_', 1)[0]);
```

Comment:
@alinedoleron, considering https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Examples, do we need to keep the `[0]`?

Reply:
Yes, because `x.split('_', 1)` returns an array with one element. I could use instead of [0] the x.split('_1).join('') if you prefer.